python_client/setup.py

```diff
@@ -0,0 +1,40 @@
+from setuptools import find_packages, setup
+
+
+setup(
+    name='miqa_python_client',
+    version='0.1.0',
+    description='A python library for interacting with a MIQA API',
+    long_description_content_type='text/markdown',
+    license='Apache 2.0',
+    author='Kitware, Inc.',
+    author_email='kitware@kitware.com',
+    keywords='',
+    classifiers=[
+        'Development Status :: 3 - Alpha',
+        'Environment :: Web Environment',
+        'Framework :: Django :: 3.0',
+        'Framework :: Django',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: Apache Software License',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python',
+    ],
+    python_requires='>=3.8',
+    packages=find_packages(),
+    include_package_data=True,
+    install_requires=[
+        "asgiref",
+        "certifi",
+        "Django>3.2,<4.0",
+        "django-s3-file-field-client==0.3.0",
+        "djangorestframework>3",
+        "idna",
+        "pytz",
+        "requests",
+        "sqlparse",
+        "urllib3",
```

Comment:
Reduce this to just the (unpinned) set of libraries that we directly depend on, which looks like just `requests` and `django-s3-file-field`.

Reply:
Changed in [426a913](https://github.com/OpenImaging/miqa/pull/533/commits/426a9131d375705a0b135b63df6920d1a73bcf76)